src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-linkedit.js

```diff
@@ -78,24 +78,24 @@ export default class EzBtnLinkEdit extends Component {
                 ReactDOM.unmountComponentAtNode(udwContainer);
             };
             const title = Translator.trans(/*@Desc("Select content")*/ 'link_edit_btn.udw.title', {}, 'alloy_editor');
-
-            ReactDOM.render(
-                React.createElement(
-                    eZ.modules.UniversalDiscovery,
-                    Object.assign(
-                        {
-                            onConfirm: udwOnConfirm,
-                            onCancel: () => ReactDOM.unmountComponentAtNode(udwContainer),
-                            title,
-                            multiple: false,
-                            startingLocationId: window.eZ.adminUiConfig.universalDiscoveryWidget.startingLocationId,
-                            restInfo: { token, siteaccess },
-                        },
-                        config
-                    )
-                ),
-                udwContainer
+            const alloyEditorCallbacks = eZ.ezAlloyEditor.callbacks;
+            const mergedConfig = Object.assign(
+                {
+                    onConfirm: udwOnConfirm,
+                    onCancel: () => ReactDOM.unmountComponentAtNode(udwContainer),
```

Comment:
I wonder, should we pass this in config even if we are using `alloyEditorCallbacks.openUdw`? Probably there is no need to unmount component at `udwContainer` in this case. What do you think?

Reply:
This is for a more generic solution. Someone can set own callback to openUDW and may rely on this method.